webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -137,275 +274,306 @@ class EventConfigComponent extends Component {
                   readOnly
                   type="text"
                   className={"form-control-plaintext readonly"}
-                  id="organisation_id"
-                  value={updatedEvent.organisation_name}
+                  id="organisation_name"
+                  name="organisation_name"
+                  value={this.props.organisation.name}
                 />
               </div>
             </div>
 
             <div className={"form-group row"}>
-              <label className={"col-sm-2 col-form-label"} htmlFor="name">
-                {t("Event Name")}
+              <label 
+                className={"col-sm-2 col-form-label"} 
+                htmlFor="name">
+                {t("Event Name")+"*"}
               </label>
 
               <div className="col-sm-10">
-                <input
-                  onChange={e => this.updateEventDetails("name", e)}
-                  type="text"
-                  className="form-control"
+                <FormTextBox
                   id="name"
+                  name="name"
+                  type="text"
+                  placeholder={t("Name of event (e.g. Deep Learning Indaba 2023)")}
+                  required={true}
+                  onChange={e => this.updateEventDetails("name", e)}
                   value={updatedEvent.name}
                 />
               </div>
             </div>
 
             <div className={"form-group row"}>
-              <label
+              <label 
                 className={"col-sm-2 col-form-label"}
-                htmlFor="description">
-                {t("Description")}
+                htmlFor="key">
+                {t("Event Key")+"*"}
```

Comment:
Is the asterisk to indicate that it's required? If so, have a look at the CSS class we have for this:
<span className="required-indicator">*</span>
(assumption with this is that it comes before the field name, rather than after like you have it)

Reply:
I wasn't sure where to find this CSS - it doesn't appear in EventConfig.css. If I just add 'required-indicator' into the label's className, it doesn't show any asterisk so I think it may not be finding it.